bin/save_cache

```diff
@@ -37,6 +37,12 @@ if [ -z "$CACHE_BUCKET_NAME" ]; then
 	fi
 fi
 
+# Use with caution – in general it's not a good idea to overwrite a cache entry
+if [ $3 == '--force' ]; then
```

Comment:
Buildkite (or the machine on which the agent run, I guess?) threw this error/warning running this code in [this build](https://buildkite.com/wordpress-mobile/wordpress-ios/builds/1681#f18e48c4-ffdd-4b40-bb56-58ce333fec81/323-347)

```
path/to/installed/pluging/bin/save_cache: line 41: [: ==: unary operator expected
```

My guess is this can be fixed by using `[[`:

```suggestion
if [[ $3 == '--force' ]]; then
```